### PR TITLE
Updates helm values for consul-k8s 1.0.9

### PR DESCRIPTION
**Pull request by missylbytes**
### Description
Updates helm values docs to be consistent with consul-k8s 1.0.9

* [x] external facing docs updated

### CODE REVIEW

**Comment by hc-github-team-consul-core** [comment]
🤔 This PR has changes in the `website/` directory but does not have a `type/docs-cherrypick` label. If the changes are for the next version, this can be ignored. If they are updates to current docs, attach the label to auto cherrypick to the `stable-website` branch after merging.

File: website/content/docs/k8s/helm.mdx [hunk 1]
```diff
@@ -308,6 +308,8 @@ Use these links to navigate to a particular top-level stanza.
     - `secretKey` ((#v-global-gossipencryption-secretkey)) (`string: ""`) - The key within the Kubernetes secret or Vault secret key that holds the gossip
       encryption key.
 
+    - `logLevel` ((#v-global-gossipencryption-loglevel)) (`string: ""`) - Override global log verbosity level for gossip-encryption-autogenerate-job pods. One of "trace", "debug", "info", "warn", or "error".
```
Comment (im2nguyen) [inline website/content/docs/k8s/helm.mdx:311]:
```suggestion
    - `logLevel` ((#v-global-gossipencryption-loglevel)) (`string: ""`) - Override global log verbosity level for `gossip-encryption-autogenerate-job` pods. One of "trace", "debug", "info", "warn", or "error".
```

File: website/content/docs/k8s/helm.mdx [hunk 2]
```diff
@@ -381,6 +385,15 @@ Use these links to navigate to a particular top-level stanza.
 
       - `secretKey` ((#v-global-tls-cakey-secretkey)) (`string: null`) - The key within the Kubernetes or Vault secret that holds the CA key.
 
+    - `annotations` ((#v-global-tls-annotations)) (`string: null`) - This value defines additional annotations for
+      tls init jobs. This should be formatted as a multi-line string.
```
Comment (im2nguyen) [inline website/content/docs/k8s/helm.mdx:389]:
```suggestion
    - `annotations` ((#v-global-tls-annotations)) (`string: null`) - This value defines additional annotations for
      tls init jobs. Format this value as a multi-line string.
```

File: website/content/docs/k8s/helm.mdx [hunk 3]
```diff
@@ -461,6 +476,15 @@ Use these links to navigate to a particular top-level stanza.
         beta.kubernetes.io/arch: amd64
       ```
 
+    - `annotations` ((#v-global-acls-annotations)) (`string: null`) - This value defines additional annotations for
+      acl init jobs. This should be formatted as a multi-line string.
```
Comment (im2nguyen) [inline website/content/docs/k8s/helm.mdx:480]: active voice

```suggestion
    - `annotations` ((#v-global-acls-annotations)) (`string: null`) - This value defines additional annotations for
      acl init jobs. Format this value as a multi-line string.
```

File: website/content/docs/k8s/helm.mdx [hunk 4]
```diff
@@ -514,6 +538,8 @@ Use these links to navigate to a particular top-level stanza.
         -o jsonpath="{.clusters[?(@.name=='<your cluster name>')].cluster.server}"
       ```
 
+    - `logLevel` ((#v-global-federation-loglevel)) (`string: ""`) - Override global log verbosity level for the create-federation-secret-job pods. One of "trace", "debug", "info", "warn", or "error".
```
Comment (im2nguyen) [inline website/content/docs/k8s/helm.mdx:541]:
```suggestion
    - `logLevel` ((#v-global-federation-loglevel)) (`string: ""`) - Override global log verbosity level for the `create-federation-secret-job` pods. One of "trace", "debug", "info", "warn", or "error".
```

File: website/content/docs/k8s/helm.mdx [hunk 5]
```diff
@@ -2199,6 +2235,8 @@ Use these links to navigate to a particular top-level stanza.
   - `enabled` ((#v-ingressgateways-enabled)) (`boolean: false`) - Enable ingress gateway deployment. Requires `connectInject.enabled=true`
     and `client.enabled=true`.
 
+  - `logLevel` ((#v-ingressgateways-loglevel)) (`string: ""`) - Override global log verbosity level for ingress-gateways-deployment pods. One of "trace", "debug", "info", "warn", or "error".
```
Comment (im2nguyen) [inline website/content/docs/k8s/helm.mdx:2238]:
```suggestion
  - `logLevel` ((#v-ingressgateways-loglevel)) (`string: ""`) - Override global log verbosity level for `ingress-gateways-deployment` pods. One of "trace", "debug", "info", "warn", or "error".
```

File: website/content/docs/k8s/helm.mdx [hunk 6]
```diff
@@ -2031,10 +2065,12 @@ Use these links to navigate to a particular top-level stanza.
 - `meshGateway` ((#v-meshgateway)) - [Mesh Gateways](https://developer.hashicorp.com/consul/docs/connect/gateways/mesh-gateway) enable Consul Connect to work across Consul datacenters.
 
   - `enabled` ((#v-meshgateway-enabled)) (`boolean: false`) - If [mesh gateways](https://developer.hashicorp.com/consul/docs/connect/gateways/mesh-gateway) are enabled, a Deployment will be created that runs
-    gateways and Consul Connect will be configured to use gateways.
+    gateways and Consul service mesh will be configured to use gateways.
     This setting is required for [Cluster Peering](https://developer.hashicorp.com/consul/docs/connect/cluster-peering/k8s).
     Requirements: consul 1.6.0+ if using `global.acls.manageSystemACLs``.
 
+  - `logLevel` ((#v-meshgateway-loglevel)) (`string: ""`) - Override global log verbosity level for mesh-gateway-deployment pods. One of "trace", "debug", "info", "warn", or "error".
```
Comment (im2nguyen) [inline website/content/docs/k8s/helm.mdx:2072]:
```suggestion
  - `logLevel` ((#v-meshgateway-loglevel)) (`string: ""`) - Override global log verbosity level for `mesh-gateway-deployment` pods. One of "trace", "debug", "info", "warn", or "error".
```

**Review by im2nguyen — APPROVED** [review]

**Comment by missylbytes** [comment]
Removed backport label @im2nguyen since this is a PR directly into the release/1.14.x branch

**Review by curtbushko — APPROVED** [review]

**Pull request merged**